CONTRIBUTING.md

```diff
@@ -63,7 +63,7 @@ Should you have any questions please contact the secretaries on info [at] php-fi
 [dot] org. Failure to comply with guidelines will result in revokation of merge
 access. Merge access is a privilege and not a right.
 
-# Tagging
+## Tagging
```

Comment:
All the fixes on this file where due to (logical) level errors. This cannot be automated, it's due to the content, not the form.

Reply:
Oh right, i never though this would be the issue, this is actually a cool fix. It does that because there can only be one `# title` .